Attempt to generate code for anonymous structs

For example:

```
/tmp/chimera_output/_ZN4dart3gui12jitter_pointE.cpp:11:33: error: unexpected parenthesis after '::'
.def_readwrite("x", &dart::gui::(anonymous struct)::x)
```

--- a/src/visitor.cpp
+++ b/src/visitor.cpp
@@ -97,6 +97,10 @@ std::string ConstructBindingName(
     if (node.IsNull() && node["name"])
         return node.as<std::string>();
 
+    // If this is an anonymous struct, then use the name of its typedef.
+    if (TypedefNameDecl *typedef_decl = decl->getTypedefNameForAnonDecl())
+        return typedef_decl->getNameAsString();
+
     // If the class is not a template class, use the unqualified string name.
     if (!isa<ClassTemplateSpecializationDecl>(decl))
         return decl->getNameAsString();

